static/templates/zgram-box.handlebars

```diff
@@ -0,0 +1,22 @@
+<div zid="{{id}}" id="{{id}}" class="messagebox">
```

Comment:
I was thinking we should keep this separated from `message-*` as I think that this would finally shape very differently from a message row (I know that this is just a prototype but later it would create problem with styling to make changes).

Reply:
I agree. I'll remove all the existing dependencies.